genenetwork/genenetwork3
# Add rif tests

### Description

This PR introduces RDF unit test cases for the RIF fetches.  For more context read [gn3/ADR-000](https://issues.genenetwork.org/topics/ADR/gn3/000-add-test-cases-for-rdf).

### Testing

- Make sure you have a local virtuoso server running.
- Make sure you set the following in your secrets file:
 ```
SPARQL_USER = "XXX"
SPARQL_PASSWORD = "XXXXX"
SPARQL_AUTH_URI="http://localhost:9082/sparql-auth/"
SPARQL_CRUD_AUTH_URI="http://localhost:9082/sparql-graph-crud-auth"
 ```
 - Run tests by: `pylint -k rdf`
 
 TODO:
- Update the CD linux container with above params.
